.github/workflows/cpu-check.yml

```diff
@@ -0,0 +1,37 @@
+name: cpu-test
+on:
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+concurrency:
+    group: ${{ github.event_name }}-${{ github.ref }}-cpu-test
+    cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  cpu-test:
+    name: cpu-test
+    runs-on: ubuntu-latest
+    env:
+      CARGO_NET_GIT_FETCH_WITH_CLI: "true"
+      RUST_MIN_STACK: 4000000 # This avoids a stack overflow when running tests
+      RUSTFLAGS: "-C target-cpu=native"
+
+    container:
+      image: rust:latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
Please add `lfs` option here and to the other job. I have noticed there is a test lfs file.

Reply:
I missed that, thanks. Adding it now.